src/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from "redux";
+import rootReducer from "./redux/reducers";
+import thunk from "redux-thunk";
+
+const configureStore = createStore(rootReducer, applyMiddleware(thunk));
+
+export default configureStore;

```

Comment:
What does `configure` mean in this naming? We usually use `store` because function named `createStore` should create instance of `store`